### PR TITLE
[NAN-457] document script model introspection endpoint

### PR DESCRIPTION
**Pull request by khaliqgant**
## Describe your changes
Alias flow/configs to `scripts/config` to better naming and update docs

## Issue ticket number and link
NAN-457

## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: 
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is:

### CODE REVIEW

File: docs-v2/reference/api/scripts/config.mdx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+title: 'Get Scripts Config'
```
Comment (bastienbeurier) [inline docs-v2/reference/api/scripts/config.mdx]: Only the first word should be capitalized (for consistency).

File: docs-v2/reference/sdks/node.mdx [hunk 2]
```diff
@@ -492,6 +492,187 @@ await nango.deleteConnection('<INTEGRATION-ID>', 'CONNECTION-ID');
 
 Empty response.
 
+# Scripts
+
+
+### Get Scripts Config
```
Comment (bastienbeurier) [inline docs-v2/reference/sdks/node.mdx]: Only the first word should be capitalized (for consistency).

File: docs-v2/reference/api/scripts/config.mdx [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+openapi: 'GET /scripts/config'
+---
```
Comment (bastienbeurier) [inline docs-v2/reference/api/scripts/config.mdx:4]: The page shows up empty for me for some reason! Did you test it locally?

Comment (khaliqgant) [reply docs-v2/reference/api/scripts/config.mdx:4]: I did and saw the same but quite a few other pages show up empty for me locally as well so wasn’t sure what was expected locally actually

Comment (bastienbeurier) [reply docs-v2/reference/api/scripts/config.mdx:4]: Oh that's right. Then let's push it and see if it shows!

**Review by bastienbeurier — APPROVED** [review]

**Pull request merged**